nemo/src/io/format_builder.rs

```diff
@@ -407,9 +441,77 @@ impl ImportExportBuilder {
             }
         }?;
 
+        let resource_builder = inner
+            .override_resource_builder(direction)
+            .or(resource_builder);
+
+        let resource = {
+            if let Some(mut rb) = resource_builder {
+                // Unpack Http-Parameters and add them to the ResourceBuilder
+                // Header-Parameters
+                let headers = parameters
+                    .get_optional(StandardParameter::HttpGetParameters.into())
+                    .map(ResourceBuilder::unpack_header_parameter);
+
+                if let Some(iter) = headers {
+                    for (key, value) in iter {
+                        rb.header_mut(key, value)
+                            .map_err(|err| builder.report_error(origin, err.into()))
+                            .ok()?;
+                    }
+                }
+
+                // GET-Parameters
```

Comment:
At least get rid of the hyphen and downcase parameters, better get rid of the comment.